lib/kwtool/accounts.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Kwtool.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+  import Ecto.Query, warn: false
+
+  alias Kwtool.Accounts.Schema.UserSchema
+  alias Kwtool.Repo
+
+  @doc """
+  Returns the list of users.
+
+  ## Examples
+
+      iex> list_users()
+      [%User{}, ...]
+
+  """
```

Comment:
Not sure if you plan to use `doctest` everywhere. Usually, we don't. But if you don't plan to use them, then removing it would be better.

Reply:
I removed it.